src/libstd/fs.rs

```diff
@@ -436,19 +436,19 @@ impl OpenOptions {
     /// Note that setting `.write(true).append(true)` has the same effect as
     /// setting only `.append(true)`.
     ///
-    /// For most filesystems the operating system guarantees all writes are
+    /// For most filesystems, the operating system guarantees all writes to be
```

Comment:
An alternative is "the operating system guarantees *that* all writes are atomic".